From 713425e00b060d05525b28639a37fa516f2a2f52 Mon Sep 17 00:00:00 2001
From: Pavan Nikhilesh <pbhagavatula@marvell.com>
Date: Thu, 25 Jul 2024 13:10:19 +0530
Subject: [PATCH 409/513] ci: update event perf numbers

Update event perf reference numbers.

Signed-off-by: Pavan Nikhilesh <pbhagavatula@marvell.com>
Change-Id: Id6641897bcf45ce8807f3566a2514268bd4f1388
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/dataplane/dpdk/+/132020
Base-Builds: sa_ip-toolkits-Jenkins <sa_ip-toolkits-jenkins@marvell.com>
Base-Tests: sa_ip-toolkits-Jenkins <sa_ip-toolkits-jenkins@marvell.com>
Tested-by: sa_ip-toolkits-Jenkins <sa_ip-toolkits-jenkins@marvell.com>
Reviewed-by: Jerin Jacob <jerinj@marvell.com>
---
 .../event_perf/ref_numbers/cn106xx_rclk2500_sclk1100.csv        | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/marvell-ci/test/cnxk-tests/event_perf/ref_numbers/cn106xx_rclk2500_sclk1100.csv b/marvell-ci/test/cnxk-tests/event_perf/ref_numbers/cn106xx_rclk2500_sclk1100.csv
index 0a51a6c6efaf9..18d70cd1decb6 100644
--- a/marvell-ci/test/cnxk-tests/event_perf/ref_numbers/cn106xx_rclk2500_sclk1100.csv
+++ b/marvell-ci/test/cnxk-tests/event_perf/ref_numbers/cn106xx_rclk2500_sclk1100.csv
@@ -28,7 +28,7 @@ cores, RESULT
 ------------------------------------------------------------------------
 GW_MODE_NO_PREF (atomic queue) output in packets per second
 cores, RESULT
-1, 5.087
+1, 4.829
 8, 31.616
 16, 61.556
 
-- 
2.25.1

